src/sig_stfl/sig_stfl.h

```diff
@@ -338,7 +343,8 @@ typedef struct OQS_SIG_STFL_SECRET_KEY {
 	 * Callback function used to securely store key data
 	 * @param[in] sk_buf The serialized secret key data to secure store
 	 * @param[in] buf_len length of data to secure
-	 * @param[in] context aids the secure writing of data
+	 * @param[in] context application supplied data used to locate where this secret key
```

Comment:
How does the application learn how much memory is to be allocated for this?

Reply:
This called by libOQS to advance the secret key index so the the next key is used for the next sign op.

Reply:
This function pointer points to the application's secure storage function. It is called by the oqs library. So the buffer size will be known then.